webapp/src/app/files/file.service.ts

```diff
@@ -29,6 +29,18 @@ export class FileService implements OnDestroy {
     this.subscriptions.forEach((subscription: Subscription) => subscription.unsubscribe());
   }
 
+  refresh() {
+    this.subscriptions.push(
+      this.containerApi.getContainers().subscribe(
+        (containers: Container[]) => {
+          this.containers = containers;
+          this.setFilesToContainers();
+        },
+        (err) => {
+          console.log(err);
```

Comment:
are you sure?  the uiService is not currently injected into file.service... plus, this is not a user generated action